routes/opportunity.js

```diff
@@ -24,11 +24,16 @@ const opportunityManager = new OpportunityManager(),
  *    responses:
  *      200:
  *        description: document the API response on success here
- *            
+ *
  */
-router.post('/', (req, res) => {
-  console.log('Inside Route');
-  opportunityController.createOpportunity(req, res);
-});
+router
+  .get('/', (req, res) => {
+    console.log('Inside GET Route');
+    opportunityController.getOpportunities(req, res);
+  })
+  .post('/', (req, res) => {
+    console.log('Inside Route');
+    opportunityController.createOpportunity(req, res);
+  });
```

Comment:
please document the API endpoint using Swagger. Refer to Readme for a tutorial.

Reply:
It comes under separate issue right? Sorry to ask this as a budding open source contributor, if you have this as a separate issue I can get points for GSSOC. @abdus

Reply:
Usually, API Documentation comes in handy with the same issue, but let me discuss with @abdus and get back to you on this.

Reply:
There is a separate issue #19  for Documentation for POST API on swagger . That's why asked @Manvityagi

Reply:
@rohithmsr The previous API was added as a part of the basic setup, so swagger documentation wasn't added to it. 
As per the standards in the ideal case, the person who writes the API shall also write the swagger doc for the same API. 

I am increasing the LEVEL of this API from 2 to 3, so please also commit the documentation of this endpoint using Swagger. 

Swagger is easy, You will understand it at once, if you face any problems, feel free to ping on discord channel #opportunity-calendar-backend. 

CC: @abdus

Reply:
Sure , will work on it